### PR TITLE
tests complete/ready for review

### PR DESCRIPTION
**Pull request by ekm6**

### CODE REVIEW

File: Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs [hunk 1]
```diff
@@ -28,6 +32,21 @@ public AuthServiceTester()
             _output = new TestOutputHelper();
         }
 
+        [Theory]
+        [InlineData(11, 11, true)]
+        [InlineData(2, 1, false)]
+        public void GenerateAccessTokenTest(int id1, int id2, bool expected)
```
Comment (joeldevries) [inline Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs:38]: In the method name, include what you are testing for.

```suggestion
        public void GenerateAccessToken_TestDifferentEmployeesGetDifferenTokens(int id1, int id2, bool expected)
```

File: Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs [hunk 2]
```diff
@@ -28,6 +32,21 @@ public AuthServiceTester()
             _output = new TestOutputHelper();
```
Comment (joeldevries) [inline Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs:32]: This variable does appear to be used for anything.

File: Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs [hunk 3]
```diff
@@ -28,6 +32,21 @@ public AuthServiceTester()
...
         }
 
+        [Theory]
+        [InlineData(11, 11, true)]
+        [InlineData(2, 1, false)]
+        public void GenerateAccessTokenTest(int id1, int id2, bool expected)
+        {
+            var _activeDirectoryRepository = A.Fake<IActiveDirectoryRepository>();
+            var _employeeRepository = A.Fake<IEmployeeRepository>();
+            var _jwtsettings = A.Fake<IOptions<JWTSettings>>();
```
Comment (joeldevries) [inline Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs:42]: When naming variables, use underscores to start member (class-level) variables. Just use camelCase for local variables.

File: Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs [hunk 4]
```diff
@@ -28,6 +32,21 @@ public AuthServiceTester()
...
+            _jwtsettings.Value.SecretKey = "testsecretkeysubjecttochange";
+            var sut = new AuthService(_employeeRepository, _activeDirectoryRepository, _jwtsettings);
+            string token1 = sut.GenerateAccessToken(id1);
+            string token2 = sut.GenerateAccessToken(id2);
+            Assert.Equal(expected, token1 == token2);
+        }
+
         [Fact]
         public void GenerateRefreshTokenTest()
```
Comment (joeldevries) [inline Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs:51]: You are running your tests against a Fake IAuthService, which doesn't actually run any of your code.

File: Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs [hunk 5]
```diff
@@ -38,5 +57,29 @@ public void GenerateRefreshTokenTest()
...
+            goodRequest.Username = "***put your username here***";
+            goodRequest.Password = "**put your password here**";
+            yield return new object[] { goodRequest, OperationResultType.Ok };
+        }
+
+        [Theory]
+        [MemberData(nameof(TestLoginData))]
+        public void Login(AuthRequest request, OperationResultType expected)
+        {
+            var sut = A.Fake<IAuthService>();
```
Comment (joeldevries) [inline Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs:77]: Using a fake service again here to test.

File: Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs (hunk 2 above)
Comment (ekm6) [reply Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs:32]: I was using it for debugging earlier in the process. I'll remove it

File: Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs (hunk 3 above)
Comment (ekm6) [reply Benchmark.WebApi/Benchmark.WebApi/BenchmarkTester/AuthServiceTester.cs:42]: thanks I'll keep that in mind

**Comment by charkour** [comment]
@ekm6 , should this PR be closed?

**Pull request closed without merging**